tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -308,9 +308,12 @@ describeSmokeSuite("S300", `Verifying balances consistency`, (context, testIt) =
             // which make deposit optional.
             // TODO: better handle unwrapping
             return {
-              accountId: (deposit.unwrap ? deposit.unwrap() : deposit).who.toHex(),
+              accountId: (deposit.unwrap ? deposit.unwrapOrDefault() : deposit).who.toHex(),
```

Comment:
This is not good, it will simply make the deposit with address 0x000 and value 0
We should filter those deposit before

Reply:
line 304:
`!!value || !value.isNone` I think that would work better

Reply:
Given that that the default amounts don't introduce a balance discrepancy i thought this would be valid, but youre right - `isNone` is cleaner. Done.